doc/screenshot.js

```diff
@@ -1,4 +1,5 @@
-const timeout = 120000;
+// const timeout = 120000;
+const timeout = 240000;
```

Comment:
😀

Reply:
> Wow, this really is quite the body of work. +1 +1
> 
> Are we missing a sun gear here? <img alt="image" width="1357" src="https://user-images.githubusercontent.com/9359447/161437180-7bd540e8-980d-434e-9032-cd22cb5220d0.png">
> 
> This is just a curiosity not a real concern, but any thoughts on why so many of the bounding boxes changed from 19.97 to 19.98?

I'm not sure, but I did update the bounding box code for segments, etc.

Reply:
Hmm, that's a good find -- yes, let's find out what happened to that gear.

Reply:
Ok, that gear should be fixed now: https://github.com/jsxcad/JSxCAD/blob/73291b55aa9992d97977bd02b6e4fd2e8d7d815c/nb/projects/gear/gear.md.8.png